src/main/java/com/epam/aidial/core/ProxyContext.java

```diff
@@ -36,6 +36,7 @@ public class ProxyContext {
     private final ApiKeyData apiKeyData;
     private final String spanId;
     private final String parentSpanId;
+    private final String sourceDeployment;
```

Comment:
same as Key.getProject ?

Reply:
yes. The purpose of `sourceDeployment` (deployment.getName) is the same as key.getProject